src/Components/Results/Results.js

```diff
@@ -0,0 +1,65 @@
+import React, {useState, useEffect} from 'react'
+import Card from '../Card/Card'
+import './Results.css'
+import { searchAlbums } from '../../api'
+
+function Results({query}) {
+  const [results, setResults] = useState([])
+  const [pagination, setPagination] = useState({page: 1, pages: 1})
+  const [isLoading, setLoading] = useState(true)
+
+  useEffect(() => {
+    setLoading(true)
+    searchAlbums(query, 1)
+      .then(data => {
+        setResults(data.results)
+        setPagination(data.pagination)
+        setLoading(false)
+      })
+      .catch(error => alert(error.message))
+  }, [ query ])
+
+  const handleClick = (direction) => {
+    setLoading(true)
+    if (direction === 'last') {
```

Comment:
We can save the value of `pagination.page - 1` or `pagination.page + 1` to a variable using a conditional to pass as the second argument when the `searchAlbums` function is invoked. This way you only need to call it once and you can make the code more DRY

Reply:
true, implemented this